notebooks/maintel/BLOCK-T3.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "0",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Disable hexapod compensation mode
</pre><p>Maybe remove one of the comments?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_block_utils/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>